src/dotnet/APIView/APIViewWeb/Controllers/CommentsController.cs

```diff
@@ -22,14 +22,15 @@ public CommentsController(CommentsManager commentsManager, ReviewManager reviewM
         }
 
         [HttpPost]
-        public async Task<ActionResult> Add(string reviewId, string revisionId, string elementId, string commentText)
+        public async Task<ActionResult> Add(string reviewId, string revisionId, string elementId, string commentText, string resolutionLock = "off")
```

Comment:
Do we need default value for this new param? Can't this be a boolean param?

Reply:
Due to the way HTML form checkboxes work, this will either be not passed or have a value (of the checkbox on the page).
There is one potential workaround, where there is a "default value" checked box that's hidden on the html page, if you'd rather me use that I can